### PR TITLE
split e2e test with sample test

### PR DESCRIPTION
**Pull request by IronPan**
This is part 2 of the test split
for context see https://github.com/kubeflow/pipelines/pull/799

### CODE REVIEW

File: config/jobs/kubeflow/kubeflow-presubmits.yaml [hunk 1]
```diff
@@ -199,33 +199,33 @@ presubmits:
...
         - ./test/presubmit-tests-with-pipeline-deployment.sh
         args:
         - --workflow_file
         - e2e_test_gke_v2.yaml
         - --test_result_folder
         - e2e_test
-      - name: kubeflow-pipeline-sample-test
-        image: gcr.io/k8s-testimages/kubekins-e2e:v20190205-d83780367-1.10
+
+  kubeflow/pipelines:
```
Comment (krzyzacy) [inline config/jobs/kubeflow/kubeflow-presubmits.yaml]: drop the extra `kubeflow/pipelines` key?

Comment (IronPan) [reply config/jobs/kubeflow/kubeflow-presubmits.yaml]: done

**Comment by IronPan** [comment]
tests seems hanging.

**Comment by krzyzacy** [comment]
hummmmmm?!

**Pull request merged**